content/en/blog/_posts/2024-08-15-moving-cgroup-v1-support-maintenance-mode-kubernetes-1-31.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: blog
+title: "Kubernetes 1.31: Moving cgroup v1 Support into Maintenance Mode in Kubernetes 1.31"
+date: 2024-08-15
+slug: kubernetes-1-31-moving-cgroup-v1-support-maintenance-mode
+author: Harshal Patil
+---
+
+As Kubernetes continues to evolve and adapt to the changing landscape of
+container orchestration, the community has decided to move cgroup v1 support
+into [maintenance mode](#what-does-maintenance-mode-mean) in v1.31.
+This shift aligns with the broader industry's move towards cgroup v2, offering
```

Comment:
I would go for a couple of clarity changes:

"offering improved functionalities: including scalability and a more consistent interface." - imho easier to read

"Before we dive into the consequences for Kubernetes, let's take a step back to understand what cgroups are and their significance in Linux." - avoids Kubernetes repetition and beginning a sentence with"but"

Reply:
Solved!